ACadSharp/IO/DXF/DxfStreamWriter/DxfStreamWriterBase.cs

```diff
@@ -29,13 +31,28 @@ public void Write(int code, CSMath.IVector value, DxfClassMap map)
 			}
 		}
 
-		public void WriteHandle(int code, IHandledCadObject value, DxfClassMap map)
+		public void WriteCmColor(int code, Color color, DxfClassMap map = null)
```

Comment:
I need to check this but I think that the codification for a true color in a DXF file is not the same as in DWG.